src/main/java/duke/command/FindCommand.java

```diff
@@ -0,0 +1,47 @@
+package duke.command;
+
+import duke.main.Storage;
+import duke.task.Task;
+import duke.task.TaskList;
+
+/**
+ * A class for the command for finding tasks in list using keywords
+ */
+public class FindCommand extends Command {
+
+    /** Keywords to search for in the task description*/
+    private final String keywords;
```

Comment:
If I'm not wrong, I think plural form for variable names should only be used on a collection of objects? Perhaps this could be renamed as keyPhrase?